src/ray/common/ray_config_def.h

```diff
@@ -335,3 +335,6 @@ RAY_CONFIG(int64_t, gcs_server_request_timeout_seconds, 5)
 
 /// Whether to enable multi tenancy features.
 RAY_CONFIG(bool, enable_multi_tenancy, false)
+
+/// The number of I/O worker that raylet starts.
+RAY_CONFIG(int, num_io_workers, 0)
```

Comment:
Does spilling still work if this is zero, or do you need to set it to 1?

Reply:
it will be pended. so basically 0 means disable object spilling. I do not want to set it to 1 since it will create extra workers.

Reply:
Can we start workers ondemand instead, and have this be a max cap?

Reply:
hmmm, that would mean the first round of object spilling would be pretty slow. would you expect this kind of behavior?

Reply:
The initial latency is inconsequential for real spilling workloads, and we can keep the workers around up to the max to avoid additional overhead.

Let's rename this to `max_io_workers` and set the default to 1 for now.

Reply:
ok